src/website/app/pages/Theming/index.js

```diff
@@ -1,55 +1,14 @@
 /* @flow */
-import React from 'react';
-import {
-  createStyledComponent,
-  getNormalizedValue,
-  pxToEm
-} from '../../../../library/styles';
+import React, { Fragment } from 'react';
 import Button from '../../SiteButton';
-import Heading from '../../SiteHeading';
 import Markdown from '../../Markdown';
 import VariableTable from '../../VariableTable';
+import VariableTableTitle from '../../VariableTableTitle';
 import groupedMineralTheme from './groupedMineralTheme';
 import content from './theming.md';
 
 type Props = {};
 
-const styles = {
-  table: {
-    '& table': {
-      tableLayout: 'fixed'
-    }
-  },
-  title: ({ theme }) => ({
-    margin: `0 0 ${getNormalizedValue(
-      pxToEm(21 - 12), // to mid-baseline
-      theme.SiteHeading_fontSize_3
-    )}`,
-    paddingTop: getNormalizedValue(
-      pxToEm(68), // to baseline
-      theme.SiteHeading_fontSize_3
-    ),
-
-    [theme.bp_moreSpacious]: {
-      fontSize: theme.SiteHeading_fontSize_3_wide,
-      margin: `0 0 ${getNormalizedValue(
-        pxToEm(19 - 12), // to mid-baseline
-        theme.SiteHeading_fontSize_3_wide
-      )}`,
-      paddingTop: getNormalizedValue(
-        pxToEm(80), // to baseline
-        theme.SiteHeading_fontSize_3_wide
-      )
-    }
-  })
-};
-
-const Table = createStyledComponent(VariableTable, styles.table);
-const Title = createStyledComponent(Heading, styles.title).withProps({
-  anchors: true,
-  level: 3
-});
-
 const REGEX_IS_COLOR = /^#/;
 
 const getColor = (theme, variable) =>
```

Comment:
We could probably export this and `getValue` from Tokens/index and import here, yeah?

Reply:
Or the other way around. Tokens feels more foundational to me.